plugins/pulp_python/plugins/distributors/steps.py

```diff
@@ -121,6 +133,55 @@ def _create_package_index(name, simple_path, packages):
 
             package_index.write(ElementTree.tostring(html, 'utf8'))
 
+    @staticmethod
+    def _create_metadata(name, packages):
```

Comment:
I'm coming to this without any context. So project is a pypi project, which can have multiple packages, but packages can only have one Project? If so, the relationship is indeed one project to many packages, which in pulp 3 would be a foreign key relationship from package back to project.